ui/desktop/app/components/settings-card/index.hbs

```diff
@@ -17,19 +17,12 @@
 
     {{#if (has-block 'action')}}
       <div class='action'>
-        {{! this will contain buttons }}
         {{yield to='action'}}
       </div>
     {{/if}}
 
   </div>
 
-  {{#if @description}}
```

Comment:
Just a heads up I actually did use this block, I think it makes sense to have it. What I had done to avoid another class was I modified the CSS to just bake it in as part of the headers class. 

I can handle merging these though since your PR can make it in first

Reply:
ahh ok thanks for the headsup-  I will undo this description change and not do the styles and after you merge your PR, it will have the correct margin, does that make it easy?

Reply:
I can just merge it normally, you don't have to change it back or you could just add the CSS i used:

```css
  .header-and-icon {
    display: flex;
    align-items: center;
    gap: 1rem;

    ~ * {
      padding-left: 2.8rem;
    }
  }
```

And I changed the `.content` margin to be `margin: 1rem 0;`, that way all the padding in the body is handled by the `~ *`

Reply:
thanks, added the styles